players/g4_player.py

```diff
@@ -158,27 +158,39 @@ def _reshape(
         # 2. find all cells we can move onto once we retract all cells in step 1
         # 3. find and return the overlap between cells in step 2 and target cells
         #    unoccupied by our Ameoba
-
+        
         retractable_cells = set(curr_state.periphery) - target
         occupiable_cells = find_movable_cells(
             list(retractable_cells), curr_state.periphery, curr_state.amoeba_map,
-            curr_state.bacteria, len(retractable_cells)
-        )
+            curr_state.bacteria, -1 #mini truncated the list returned #len(retractable_cells)
```

Comment:
good catch! using `-1` will cause the last element to be omitted though, so I changed the `find_movable_cell` to take in a keyword argument `n` (in place of the previously `mini`) that could be either `None` or an `int`. If `n` is not provided, the entire movable cell list will be returned, otherwise only [0:`n`] of the movable cell list will be returned.